lib/utils.ts

```diff
@@ -41,6 +41,8 @@ export enum UserType {
   DRONE = "רחפן",
 }
 
+export let shmirot = ["patrol", "flowers", "gate", "drone"];
```

Comment:
Those are already defined on the `Positions` enum in `utils`

Reply:
Positions holds the names, i need the type inorder to find the relevent type in the notion object